orc8r/gateway/python/magma/magmad/BUILD.bazel

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+ORC8R_ROOT = "../../"
+
+SRCS_LIB = [
+    "bootstrap_manager.py",
+    "config_manager.py",
+    "events.py",
+    "gateway_status.py",
+    "metrics_collector.py",
+    "metrics.py",
+    "proxy_client.py",
+    "rpc_servicer.py",
+    "service_health_watchdog.py",
+    "service_manager.py",
+    "service_poller.py",
+    "state_reporter.py",
+    "sync_rpc_client.py",
+]
+
+DEPS_MAIN = [
+    ":magmad_lib",
+    "//orc8r/gateway/python/magma/common:grpc_client_manager",
+    "//orc8r/gateway/python/magma/common:sentry",
+    "//orc8r/gateway/python/magma/magmad/generic_command:command_executor",
+    "//orc8r/gateway/python/magma/magmad/upgrade:upgrader",
+    "//orc8r/gateway/python/magma/magmad/upgrade:magma_upgrader",
+]
+
+DEPS_LIB = [
+    "//orc8r/gateway/python/magma/common:cert_utils",
+    "//orc8r/gateway/python/magma/common:rpc_utils",
+    "//orc8r/gateway/python/magma/common:sdwatchdog",
+    "//orc8r/protos:bootstrapper_python_grpc",
+    "//orc8r/gateway/python/magma/magmad/check/network_check:ping",
+    "//orc8r/gateway/python/magma/common/health:service_state_wrapper",
+    "//orc8r/swagger:magmad_events_v1",
+    "//orc8r/gateway/python/magma/common:misc_utils",
+    "//orc8r/gateway/python/magma/magmad/check/kernel_check:kernel_versions",
+    "//orc8r/gateway/python/magma/magmad/check/machine_check:cpu_info",
+    "//orc8r/gateway/python/magma/magmad/check/network_check:routing_table",
+    "//orc8r/protos:metricsd_python_grpc",
+    "//orc8r/gateway/python/magma/common:stateless_agw",
+    "//orc8r/protos:magmad_python_grpc",
+    "//orc8r/gateway/python/magma/common:cert_validity",
+    "//orc8r/protos:state_python_grpc",
+    "//orc8r/protos:sync_rpc_service_python_grpc",
+    "@aioh2_repo//:aioh2",
+    "//orc8r/gateway/python/magma/common:streamer",
+]
```

Comment:
All other lists are sorted lexicographical. Would we also want to do this for this list?

Reply:
`bazel run //:buildifier` should take care of it :)

Reply:
Sadly, no - we will sort it manually. Are you sure it should take care of it? Maybe because we moved it into variables.

Reply:
ohhh maybe that is why....

Reply:
I would opt for placing these items directly into the macro to avoid having to jump around when reading, but open to other suggestions.

Reply:
I do not have a strong opinion about that. We will change this as recommended. For existing services we will keep this in mind and change it when we touch respective code.

Reply:
(btw tested: the buildifier works when the variables are inlined)

Reply:
Change pushed.